### PR TITLE
New End Table graphics

### PR DESCRIPTION
**Pull request by TLHeart**

### CODE REVIEW

File: Mods/ModVarietyPack/Defs/ThingDefs/Plants_Cultivated.xml [hunk 1]
```diff
@@ -61,8 +61,8 @@
       <soundHarvesting>Harvest_Standard</soundHarvesting>
       <soundHarvestFinish>Harvest_Standard_Finish</soundHarvestFinish>
       <sowWork>150</sowWork>
-      <!--<fertilityMin>0.5</fertilityMin>
-      <fertilityFactorGrowthRate>1.0</fertilityFactorGrowthRate>-->
+      <fertilityMin>0.5</fertilityMin>
+      <fertilityFactorGrowthRate>1.0</fertilityFactorGrowthRate>
```
Comment (simon-82) [inline Mods/ModVarietyPack/Defs/ThingDefs/Plants_Cultivated.xml]: This won't work. 
I have them defined for each plant seperate, See the top of this file.
If the factor is 1, the boost would be too big. 
Example: If you have a plant that needs at least 2.0 fertility to grow (so at least watered soil), it would then grow with 200% by default. 
Well, now that I think about it, it is overriden by the setting in each plant, right?
Did you test this? Does it add better grass growing? Because you commented something about grass growing...

File: Mods/ModVarietyPack/Defs/ThingDefs/Buildings_Furniture.xml [hunk 2]
```diff
@@ -1888,105 +1738,14 @@
...
-      <li>
-        <compClass>CompGlower</compClass>
-        <glowRadius>6</glowRadius>
-        <glowColor>(0,0,217,0)</glowColor>
+      <li Class="CommunityCoreLibrary.CompProperties_ColoredLight">
+        <compClass>CommunityCoreLibrary.CompColoredLight</compClass>
       </li>
     </comps>
-    <researchPrerequisite>ColoredLights</researchPrerequisite>
+    <researchPrerequisite>PowerI</researchPrerequisite>
```
Comment (simon-82) [inline Mods/ModVarietyPack/Defs/ThingDefs/Buildings_Furniture.xml:1808]: What about the colored lights research then? The research project is still there, right? Does it unlock anything now? 
I guess not? What to replace it with then? Or do you suggest to just get rid of this research project altogether?
Edit: Oh sorry my bad. Colored lights is a vanilla project. I suppose the CCL designator becomes available after you research it. 
It's too bad the actual texture doesn't change with this. Only the glow effect does. 
In game it doesn't look too bad. At least inside a room, The glow effect goes over the texture, so it looks like it changed. Outside you can clearly see the texture stays the same-

File: Mods/ModVarietyPack/Defs/ThingDefs/Buildings_Furniture.xml [hunk 3]
```diff
@@ -2038,105 +1797,13 @@
         <glowRadius>3</glowRadius>
         <glowColor>(217,217,208,0)</glowColor>
       </li>
-    </comps>
-    <researchPrerequisite>PowerI</researchPrerequisite>
-  </ThingDef>
-
-  <ThingDef ParentName="FloorLightSmallStripe">
-    <defName>FloorLightSmallStripe_Red</defName>
```
Comment (simon-82) [inline Mods/ModVarietyPack/Defs/ThingDefs/Buildings_Furniture.xml]: If we keep the CCL colored lights, all colored textures need to be deleted.

File: Mods/ModVarietyPack/Defs/ThingDefs/Buildings_Furniture.xml [hunk 4]
```diff
@@ -2038,105 +1797,13 @@
...
-    <label>floor light (small stripe) red</label>
-    <graphicData>
-      <texPath>Buildings/Lights/FloorLightSmallStripe_Red</texPath>
```
Comment (simon-82) [inline Mods/ModVarietyPack/Defs/ThingDefs/Buildings_Furniture.xml:2049]: If we keep the CCL colored lights, all colored textures need to be deleted

**Comment by simon-82** [comment]
Alright. 
I merged everything of this request now into the dev branch. 
I am strongly assuming the other pull request you made is the same as this one. 
All future pull requests only to the dev branch please. 
Sorry for not setting it up before, that was my bad, so you had no choice other than merging into master. 
And again thanks for all your work so far! You are doing a really great job! Keep it up :+1:

BTW, take a look at this and see what you have done! 
lol, pretty neat :+1: [Spaghetti!!!](https://github.com/simon-82/ModVarietyPack/network)

File: Mods/ModVarietyPack/Defs/ThingDefs/Buildings_Furniture.xml [hunk 5]
```diff
@@ -661,77 +661,11 @@
         <glowRadius>12</glowRadius>
         <glowColor>(217,217,208,0)</glowColor>
       </li>
-    </comps>
-    <researchPrerequisite>PowerI</researchPrerequisite>
-  </ThingDef>
-
-  <ThingDef ParentName="StandingLampBase">
-    <defName>StandingLamp_Red</defName>
-    <label>standing lamp (red)</label>
```
Comment (simon-82) [inline Mods/ModVarietyPack/Defs/ThingDefs/Buildings_Furniture.xml:670]: There is a problem with deleting the colored standing lamps. 
As these are vanilla items, if you delete the defs here (they are overriding the vanilla colored standing lamps), they get added back to the furniture category with the vanilla textures. 
So at least the standing lamps must be kept.

File: Mods/ModVarietyPack/Defs/ThingDefs/Items_Meals.xml [hunk 6]
```diff
@@ -222,13 +222,16 @@
     </graphicData>
     <equippedAngleOffset>-150</equippedAngleOffset>
     <rotatable>false</rotatable>
-    <stackLimit>25</stackLimit>
```
Comment (simon-82) [inline Mods/ModVarietyPack/Defs/ThingDefs/Items_Meals.xml:225]: Oh noes! 
One beer less per stack? Why you take away my beer!? 
Now I am sad :(

File: Mods/ModVarietyPack/Defs/ThingDefs/Buildings_Furniture.xml (hunk 2 above)
Comment (TLHeart) [inline Mods/ModVarietyPack/Defs/ThingDefs/Buildings_Furniture.xml:1808]: I did not remove the colored lights research from the base lights, look at the line numbers. So the texture does not change, the color does. And you can add more colors if you want, without having to create complete textures, and more code. It is your project use if you want , it makes the lighting menu so much less cluttered.

File: Mods/ModVarietyPack/Defs/ThingDefs/Plants_Cultivated.xml (hunk 1 above)
Comment (TLHeart) [inline Mods/ModVarietyPack/Defs/ThingDefs/Plants_Cultivated.xml]: Actually this was not to come through... Was testing to see if the native grass would grow. it still does not. Playing in a temperate forest, in 30 game days looks like I am playing in the arid shrublands, all the native animals are starving, which means my colonist will also be starving soon, with nothing left to hunt. 

Don't know what you changed to cause this problem, but it really needs to be found and fixed.

File: Mods/ModVarietyPack/Defs/ThingDefs/Buildings_Furniture.xml (hunk 5 above)
Comment (TLHeart) [inline Mods/ModVarietyPack/Defs/ThingDefs/Buildings_Furniture.xml:670]: I Did not catch that. Will have to look into how others remove the vanilla colored standing lamps.

File: Mods/ModVarietyPack/Defs/ThingDefs/Items_Meals.xml (hunk 6 above)
Comment (TLHeart) [inline Mods/ModVarietyPack/Defs/ThingDefs/Items_Meals.xml:225]: a case of beer is 24. :)

File: Mods/ModVarietyPack/Defs/ThingDefs/Buildings_Furniture.xml (hunk 5 above)
Comment (simon-82) [inline Mods/ModVarietyPack/Defs/ThingDefs/Buildings_Furniture.xml:670]: Only if you remove the def in Core. But editing defs in Core is bad. 
The defName is unique, so if it not overriden it will always be there

File: Mods/ModVarietyPack/Defs/ThingDefs/Buildings_Furniture.xml (hunk 2 above)
Comment (simon-82) [inline Mods/ModVarietyPack/Defs/ThingDefs/Buildings_Furniture.xml:1808]: Yep, the CCL lights are better, much less clutter in the menu, you are right. I will keep it.

File: Mods/ModVarietyPack/Defs/ThingDefs/Buildings_Furniture.xml (hunk 5 above)
Comment (TLHeart) [inline Mods/ModVarietyPack/Defs/ThingDefs/Buildings_Furniture.xml:670]: I never touch the core. That was why I asked about you having the core file in your modpack... but you are adding to it.

Comment (simon-82) [inline Mods/ModVarietyPack/Defs/ThingDefs/Buildings_Furniture.xml:670]: Yep, never touch the Core :stuck_out_tongue: 

So will you just re-add the lines you deleted and send me another pull request? Or should I do it? The way it is now is no good.

File: Mods/ModVarietyPack/Defs/ThingDefs/Items_Meals.xml (hunk 6 above)
Comment (simon-82) [inline Mods/ModVarietyPack/Defs/ThingDefs/Items_Meals.xml:225]: That totally makes sense!

File: Mods/ModVarietyPack/Defs/ThingDefs/Apparel_MVP.xml [hunk 7]
```diff
@@ -2952,10 +2952,10 @@ And the surgical mask prevents unwanted bacteria entering the vic.. patient</des
 
 
 
-  <!--ArchNem© Ski™ goggles-->
+  <!--ArchNem� Ski� goggles-->
   <ThingDef ParentName="ApparelBase">
     <defName>Apparello_Brimsk</defName>
-    <label>ArchNem© Ski™ goggles</label>
+    <label>ArchNem� Ski� goggles</label>
```
Comment (simon-82) [inline Mods/ModVarietyPack/Defs/ThingDefs/Apparel_MVP.xml:2958]: Something happened here with the © symbol after your commit. 
Make sure you save in UTF-8 not something else. It gets messed up in game otherwise. 
My guess is that was because the .gitattributes messing with the text format. But not sure

File: Mods/ModVarietyPack/Defs/ThingDefs/Plants_Cultivated.xml (hunk 1 above)
Comment (simon-82) [inline Mods/ModVarietyPack/Defs/ThingDefs/Plants_Cultivated.xml]: I am working on it. 
Did you try it with 1.12? 
Also playing in temperate, I felt it was much better. For you too? Or just as bad as before?

File: Mods/ModVarietyPack/Defs/ThingDefs/Apparel_MVP.xml (hunk 7 above)
Comment (TLHeart) [inline Mods/ModVarietyPack/Defs/ThingDefs/Apparel_MVP.xml:2958]: strange, as I have it set UTF-8, but if you look at the first line it is replacing it.

File: Mods/ModVarietyPack/Defs/ThingDefs/Plants_Cultivated.xml (hunk 1 above)
Comment (TLHeart) [inline Mods/ModVarietyPack/Defs/ThingDefs/Plants_Cultivated.xml]: It is improved, grass does grow, but there is a lot of dirt, and yes this is with 1.12.

**Comment by simon-82** [comment]
Thanks for the new pull request :+1: 
I merged this one manually into the dev branch, closed
